pandas/tests/indexing/test_indexing.py

```diff
@@ -868,6 +868,13 @@ def test_no_reference_cycle(self):
         del df
         assert wr() is None
 
+    def test_label_indexing_on_nan(self):
+        # GH 32431
+        df = pd.Series([1, "{1,2}", 1, None])
+        vc = df.value_counts(dropna=False)
+        print(vc.loc[np.nan])
```

Comment:
Rather than just printing you should construct the expected result and use `tm.assert_series_equal(result, expected)` - you should see this idiom throughout the test modules if you need a reference

Reply:
Thanks for the feedback! I constructed the expected result and expected an `int`. So I compared the results just with a normal assert, since I couldn't find a method on tm to assert ints. Please let me know if this satisfies your request